### PR TITLE
Add Skyhook Kit

### PR DESCRIPTION
**Pull request by sgalat-skyhook**

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -58,3 +58,6 @@
 [submodule "kits/radar-kit"]
 	path = kits/radar-kit
 	url = git@github.com:mparticle-integrations/mparticle-android-integration-radar.git
+[submodule "kits/skyhook-kit"]
+	path = kits/skyhook-kit
+	url = git@github.com:SkyhookWireless/mparticle-android-integration-skyhook.git
```
Comment (samdozor) [inline .gitmodules]: this should point to our org.

**Comment by sgalat-skyhook** [comment]
@samdozor Of course. Skyhook SDK is not available from the Maven repository though, only from the Skyhook website. So to update the kits table in README should I put a link to the web page where the SDK can be downloaded?

**Comment by samdozor** [comment]
@sgalat-skyhook the kit code will be available via maven - that's the artifact that's listed in the README.

**Comment by sgalat-skyhook** [comment]
Oh right.

**Comment by sgalat-skyhook** [comment]
@samdozor 
Just to confirm: this build failure is expected until the Skyhook Kit impl itself is merged in, right?

Did you have a chance yet to look at the PR?
https://github.com/mparticle-integrations/mparticle-android-integration-skyhook/pull/1

Thanks!

**Comment by sgalat-skyhook** [comment]
Rebased on top of 4.13.0.

**Pull request merged**